content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -31,14 +31,14 @@ in the Kubernetes source directory for a canonical example.
 
 Package your scheduler binary into a container image. For the purposes of this example,
 you can use the default scheduler (kube-scheduler) as your second scheduler.
-Clone the [Kubernetes source code from GitHub](https://github.com/kubernetes/kubernetes)
-and build the source.
+Clone the [Kubernetes source code from GitHub](https://github.com/kubernetes/kubernetes) with the exact same branch as per the kubernetes cluster version in use and build the source.
 
 ```shell
-git clone https://github.com/kubernetes/kubernetes.git
+git clone --branch="version" https://github.com/kubernetes/kubernetes.git
 cd kubernetes
 make
 ```
+Where "version" can be for example "v1.25.4". Run `kubectl version` command to find your kubernetes cluster version in use.
```

Comment:
AFAICT, `v1.25.4` is a tag, not a branch. The branch name is something like `release-1.25`.
Please be concise on this.

Reply:
Hello @tengqm 
Yes, `v1.25.4` is a tag. `--branch` option takes tag as a value. I have mentioned "version" there because user can add the kubernetes version that they need to clone. The version that we get from 'kubectl version' command is same as that of tag in terms of git repo.